driver/network_driver.go

```diff
@@ -61,6 +62,11 @@ func (d NetworkDriver) GetCapabilities() (*network.CapabilitiesResponse, error)
 func (d NetworkDriver) CreateNetwork(request *network.CreateNetworkRequest) error {
 	logutils.JSONMessage("CreateNetwork", request)
 
+	knownOpts := map[string]bool{
+		"com.docker.network.generic":     true,
+		"com.docker.network.internal":    true,
+		"com.docker.network.enable_ipv6": true}
+
 	genericOpts, ok := request.Options["com.docker.network.generic"]
```

Comment:
This overlaps with the new code you added starting on line 98. I don't think you need both bits of code.